app/train.py

```diff
@@ -0,0 +1,16 @@
+from spacy.cli import train
+import os.path
+
+current_dir = os.path.dirname(__file__)
+
+no_entities=True
```

Comment:
Follow PEP8 - Whitespace in Expression and statements (https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements)